playground/frontend/playground_components/lib/src/constants/colors.dart

```diff
@@ -36,45 +36,51 @@ class BeamColors {
 
 class BeamGraphColors {
   static const node = BeamColors.grey3;
-  static const border = Color(0xFF45454E);
+  static const border = Color(0xff45454E);
```

Comment:
Is this deliberate? Why using lowercase for alpha and uppercase for others?

Reply:
It is for a visual separation of alpha and color code.